website/docs/SetupTypes.md

```diff
@@ -267,13 +240,13 @@ Here is the same example from above, written as a test spec and executed by WDIO
 
 ```js
 describe('DuckDuckGo search', () => {
-    it('searches for WebdriverIO', () => {
-        browser.url('https://duckduckgo.com/')
+    it('searches for WebdriverIO', async () => {
+        await browser.url('https://duckduckgo.com/')
 
-        $('#search_form_input_homepage').setValue('WebdriverIO')
-        $('#search_button_homepage').click()
+        await $('#search_form_input_homepage').setValue('WebdriverIO')
+        await $('#search_button_homepage').click()
 
-        const title = browser.getTitle()
+        const title = await browser.getTitle()
         console.log('Title is: ' + title)
         // outputs: "Title is: WebdriverIO (Software) at DuckDuckGo"
     })
```

Comment:
The paragraph below this needs to be updated or removed

Reply:
good catch!